website/docs/admin-mode.md

```diff
@@ -0,0 +1,194 @@
+
+In order to use Admin Mode, you just have to create a ServiceAccount in the _admin_ or so-called _chaos_ namespace (`litmus` itself can be used), which is tied to a ClusterRole that has the permissions to perform operations on Kubernetes resources involved in the selected experiments across namespaces. Provide this ServiceAccount in ChaosEngine's .spec.chaosServiceAccount.
+
+### Supported Experiments in Namespaced Mode
+
+| Experiment                                                                           | Scope of the experiment |
+| ------------------------------------------------------------------------------------ | ----------------------- |
+| [Pod Delete](https://docs.litmuschaos.io/docs/pod-delete/)                           | Pod Level               |
+| [Container Kill](https://docs.litmuschaos.io/docs/container-kill/)                   | Pod Level               |
+| [Pod Network Loss](https://docs.litmuschaos.io/docs/pod-network-loss/)               | Pod Level               |
+| [Pod Network Latency](https://docs.litmuschaos.io/docs/pod-network-latency/)         | Pod Level               |
+| [Pod Network Duplication](https://docs.litmuschaos.io/docs/pod-network-duplication/) | Pod Level               |
+| [Pod Network Corruption](https://docs.litmuschaos.io/docs/pod-network-corruption/)   | Pod Level               |
+| [Pod CPU Hog](https://docs.litmuschaos.io/docs/pod-cpu-hog/)                         | Pod Level               |
+| [Pod Memory Hog](https://docs.litmuschaos.io/docs/pod-memory-hog/)                   | Pod Level               |
+| [Disk Fill](https://docs.litmuschaos.io/docs/disk-fill/)                             | Pod Level               |
+| [Pod Autoscaler](https://docs.litmuschaos.io/docs/pod-autoscaler/)                   | Pod Level               |
+| [Node CPU Hog](https://docs.litmuschaos.io/docs/node-cpu-hog/)                       | Infra Level             |
+| [Node Memory Hog](https://docs.litmuschaos.io/docs/node-memory-hog/)                 | Infra Level             |
+| [Node IO Stress](https://docs.litmuschaos.io/docs/node-io-stress/)                   | Infra Level             |
+| [Node Drain](https://docs.litmuschaos.io/docs/node-drain/)                           | Infra Level             |
+| [Node Taint](https://docs.litmuschaos.io/docs/node-taint/)                           | Infra Level             |
+| [Node Restart](https://docs.litmuschaos.io/docs/node-restart/)                       | Infra Level             |
+| [Kubelet Service Kill](https://docs.litmuschaos.io/docs/kubelet-service-kill/)       | Infra Level             |
+| [Docker Service Kill](https://docs.litmuschaos.io/docs/docker-service-kill/)         | Infra Level             |
+| [EC2 instance Terminate](https://docs.litmuschaos.io/docs/ec2-terminate/)            | Infra Level             |
+| [EBS Loss](https://docs.litmuschaos.io/docs/ebs-loss/)                               | Infra Level             |
```

Comment:
1. If we can use HTML table syntax instead of markdown table syntax that will be better for all tables.
OR simply you can mention like this as well up to you.
- [EBS Loss](https://docs.litmuschaos.io/docs/ebs-loss/)  - Infra Level

2. IMO this is fine as of now but is very dynamic and we have to update it for every new experiment. So instead we can have a subsection somewhere called infra chaos and pod level chaos -- and we can provide the link for those sections only.